README.md

```diff
@@ -1,40 +1,53 @@
 # connect-payment-integration-template [**Early Access**]
+
```

Comment:
My markdown formatter installed in my IDE auto formatted the `README.md` automatically to make it more of a "officially" correctly formatted markdown file. You [can see it here](https://github.com/commercetools/connect-payment-integration-template/blob/joey-koster-ct/SCC-2569-add-manage-view-types-scopes-to-docs/connect.yaml) and compare it to the current one.

Let me know if this is undesired and I can revert it back.